app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.fragment.app.FragmentContainerView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/navigationHost"
+    android:name="androidx.navigation.fragment.NavHostFragment"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    app:defaultNavHost="true"
+    tools:context=".MainActivity"
+    app:navGraph="@navigation/nav_graph_onboarding">
```

Comment:
Since there is nothing in the body of FragmentContainerView , you should close the first tag directly
```
<androidx.fragment.app.FragmentContainerView 
    ...
    tools:context=".MainActivity"
    app:navGraph="@navigation/nav_graph_onboarding" />
```